_posts/2016-09-23-back-to-school.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Back to School with Hydra-in-a-Box
+author: Hannah Frost, Gary Geisler
+---
+
+*First in a series of blog posts highlighting the personas produced in our design process, each representing a typical user of Hydra-in-a-Box and embodying a number of use cases that our repository product, now in development, or the hosted service, now in planning, aim to fulfill.*
+
+The students return to campus this week for the start of Stanford’s fall quarter. It’s a busy time at the libraries, as staff gear up to engage with faculty and students in support of their courses, research and related academic pursuits. The digital repository plays an increasingly essential role in supporting these activities, too. Our persona featuring Aaron Maddox, a 2nd year Master’s student at a small private university, demonstrates this point clearly. This portrait also shows how the Hydra-in-a-Box repository can serve as both an institutional repository as well as a repository for the library’s digital collections.
+(You can download
+<a href="https://wiki.duraspace.org/download/attachments/76841397/Hydra-in-a-Box%20Personas.pdf?version=3&modificationDate=1465782652119&api=v2">all six of the Hydra-in-a-Box personas</a>
+and other
+<a href="https://wiki.duraspace.org/display/hydra/Hydra-in-a-Box+Design+Documents">project design documents</a>
+from the DuraSpace wiki.)
+
+Aaron uses the repository for three distinct purposes. His first goal is to discover and access material for his research. The scenario describes how he is able to accomplish this leveraging the power of a Blacklight/Solr-driven front end to find and download material, easily determining its rights status due to the standardized rights statement displayed in the metadata, and capturing a formatted citation that works with common citation management tools.
+
+<figure class="image fit persona">
+  <img src="/images/Hydra-in-a-Box_Persona-Student.png" alt="Student Persona" />
```

Comment:
Can we wrap the image in a link to the image, so people can click through if they'd like? (This might mitigate the concerns @ggeisler mentioned around the addition of a JS modal for zoom for the time being.)